doc/man3/OPENSSL_malloc.pod

```diff
@@ -136,7 +136,9 @@ This adds some overhead by keeping a list of all memory allocations, and
 removes items from the list when they are free'd.
 This is most useful for identifying memory leaks.
 CRYPTO_set_mem_debug() turns this tracking on and off.  It is normally
-called at startup, but can be called at any time.
+called at startup, but in order to have any effect it must be called
+before any of the allocation functions (e.g., CRYPTO_malloc()) are done,
+including libraries internals.
```

Comment:
So essentially, this is saying "It is normally called at startup, but you damn make sure to call it real early, ya hear?"

While that's fine as a message, the "this, but that" phrasing seems a bit odd, so if you could rephrase that, I think it would be for the best.